docs/src/apireference.md

```diff
@@ -677,11 +677,14 @@ Utilities.operate!
 Utilities.vectorize
 ```
 
-## Set utilities
+## Constraint utilities
 
-The following utilities are available for sets:
+The following utilities are available for moving the function constant to the
+set for scalar constraints:
 ```@docs
 Utilities.shift_constant
+Utilities.add_scalar_constraint
```

Comment:
Sorry for the bikeshedding, but could `add_scalar_constraint` be named `normalize_and_add_constraint`?  The name `add_scalar_constraint` doesn't suggest that any normalization is happening; it sounds like a specialization of `add_constraint`.